sdl-core/src/main/scala/io/smartdatalake/definitions/Environment.scala

```diff
@@ -50,90 +51,150 @@ object Environment {
    * The environment parameter can contain multiple authorities separated by comma.
    * An authority is compared against the filesystem URI with contains(...)
    */
-  var hadoopAuthoritiesWithAclsRequired: Seq[String] = {
-    EnvironmentUtil.getSdlParameter("hadoopAuthoritiesWithAclsRequired")
-      .toSeq.flatMap(_.split(','))
+  def hadoopAuthoritiesWithAclsRequired: Seq[String] = {
+    if (_hadoopAuthoritiesWithAclsRequired.isEmpty) {
```

Comment:
I dont understand this. These changes are already on develop-spark3. I had to pull develop-spark3 because of conflicts into this branch, now this PR somehow has an incorrect diff... i'll check on another branch PR if it works better.